### PR TITLE
Add calculation functionality to calculator

### PR DESCRIPTION
**Pull request by tiagomarin**
In this pull request I added the calculation functionality for the user by:

- [x] Copying the files calculate.js and operate.js into a logic/ directory.
- [x] Installing big.js dependency.
- [x] Importing calculate.js and operate.js files in Calculator component.
- [x] Implementing event handlers for the math logic.
- [x] Testing app in the browser.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,36 +1,47 @@
...
-          <input type="button" className="btn operator" value="-" />
-          <input type="button" className="btn" value="1" />
-          <input type="button" className="btn" value="2" />
-          <input type="button" className="btn" value="3" />
-          <input type="button" className="btn operator" value="+" />
-          <input type="button" className="btn" value="0" />
-          <input type="button" className="btn" value="." />
-          <input type="button" className="btn operator" value="=" />
+          <input type="text" name="input" className="btn input" id="input" value={next || total} />
+          <input onClick={this.onClickHandler} type="button" className="btn" value="AC" />
```
Comment (codecaiine) [inline src/components/Calculator.js]: - [x] Good job so far with your project implementation it is great :+1:  But I can't perform **"AC"** operation I'm getting errors :cry: , Please fix it in order to make your calculator works as expected :smile: 

- [x] One more thing I'm getting the error in the console  **"You provided a value prop to a form field without an onChange handler" this error occurs generally when we set a value prop on a field that has no onChange handler.** I suggest you use the defaultValue prop or set an onChange prop on the field. Please fix it to make your project completed. :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Tiago-Lelinski-Marin , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good use of gitflow :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Great project implementation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Tiago-Lelinski-Marin  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (tiagomarin) [reply src/components/Calculator.js]: @codecaiine  Thanks a lot for your time!

**Pull request merged**